frontend/src/pages/Home.tsx

```diff
@@ -0,0 +1,92 @@
+import useProducts from "../hooks/useProducts";
+import ProductCard from "../components/ProductCard";
+import { Box, Typography, List, ListItem, ListItemText } from "@mui/material";
+
+const ProductsList: React.FC = () => {
+  const { products, loading, error } = useProducts();
+
+  const categories = ["Everything"];
+
+  if (loading) return <div style={{ color: "black" }}>Ładowanie...</div>;
```

Comment:
to enhance user experience we could use sth likethis: https://mui.com/material-ui/react-skeleton/ or https://mui.com/material-ui/react-progress/